back/app/serializers/web_api/v1/event_serializer.rb

```diff
@@ -1,11 +1,32 @@
 # frozen_string_literal: true
 
 class WebApi::V1::EventSerializer < WebApi::V1::BaseSerializer
-  attributes :title_multiloc, :location_multiloc, :start_at, :end_at, :created_at, :updated_at
+  attributes(
+    :title_multiloc,
+    :location_multiloc,
+    :attendees_count,
+    :start_at,
+    :end_at,
+    :created_at,
+    :updated_at
+  )
 
   attribute :description_multiloc do |object|
     TextImageService.new.render_data_images object, :description_multiloc
   end
 
   belongs_to :project
+
+  belongs_to(
+    :user_attendance,
+    record_type: WebApi::V1::Events::AttendanceSerializer.record_type,
+    serializer: WebApi::V1::Events::AttendanceSerializer
+  ) do |event, params|
+    # +:current_user_attendances+ param is used with collections to avoid N+1 queries
+    if (attendances = params[:current_user_attendances])
```

Comment:
This mechanism isn't wrong, per se, just wondering whey we don't this in the more traditional `includes` way. If it's about the parameter, we can add a scope to the model to query for attendances for a specific user and pass that to `includes` or `joins`, I think. Or is there a reason this doesn't work here?

Not blocking, but would be more in line with the conventions if there's no good reason.

Reply:
I didn't use `.includes(:attendances)` to avoid instantiating many objects, we don’t actually need and I could not figure out a way to instantiate only the attendance of the current user. If using a scope works, you're my savior :)

By the way, this pattern is employed elsewhere (which is another reason why i thought that was not possible).

Reply:
@kogre, how would you do that exactly?

The only solution I could think of looks like this:
```ruby
class Event
  has_one :current_user_attendance, -> { where(attendee: Current.user) }, class_name: 'Events::Attendance'
  # ...
end
```

However, this approach relies on a global state, and `Current.user` does not exist and needs to be implemented.

Reply:
As discussed, closing this one because we don't want to rely on global state and parameterized associations are not a thing.